ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -191,12 +230,24 @@ export const RuntimePanel = withCurrentWorkspace()(({workspace}) => {
     <FlexRow style={{justifyContent: 'flex-end', marginTop: '.75rem'}}>
       <Button
         aria-label={currentRuntime ? 'Update' : 'Create'}
-        disabled={status !== RuntimeStatus.Running || !runtimeChanged}
+        disabled={
+          !runtimeChanged
+          || status in [
```

Comment:
Would invert this predicate, i.e. `![null, Deleted, Running, Stopped].includes(status)`

This is also an incorrect usage of the `in` keyword. `x in o` checks whether `o` has key `x`, so these string checks will not pass, but something like `'includes' in []` will pass.

Reply:
i clearly have python on the brain.